src/sentry/static/sentry/app/views/releasesV2/list/releaseCard.tsx

```diff
@@ -184,22 +180,25 @@ const VersionWrapper = styled('div')`
   display: inline-block;
 `;
 
-const TimeWithIcon = styled(({date, ...props}) => (
-  <span {...props}>
-    <ClockIcon className="icon icon-clock" />
-    <TimeSince date={date} />
-  </span>
-))`
-  align-items: center;
-  color: ${p => p.theme.gray2};
+const DeployEnv = styled(Tag)`
+  background-color: ${p => p.theme.gray4};
+  color: ${p => p.theme.white};
   font-size: ${p => p.theme.fontSizeSmall};
-  display: none;
-  @media (max-width: ${p => p.theme.breakpoints[1]}) {
-    display: inline-flex;
+  margin-right: ${space(1)};
+  position: relative;
+  bottom: ${space(0.25)};
+  display: inline-block;
+  max-width: 100px;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
maybe you could use `overflowEllipsis` here

Reply:
I wanted to at first, but it sets also `display`, `width`, `white-space` and I don't want those here.